pkg/products/rhsso/prometheusRules.go

```diff
@@ -71,6 +72,77 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 					},
 				},
 			},
+
+			//SLO-availability-alerts:
+			//https://sre.google/workbook/alerting-on-slos/
+			//https://promtools.dev/alerts/errors
+			{
+				AlertName: "rhsso-slo-availability-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "rhsso-slo-availability.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: fmt.Sprintf("%sRhssoAvailability5mto1hErrorBudgetBurn", strings.ToUpper(installationName)),
+						Annotations: map[string]string{
+							"sop_url": resources.SopUrlSloRhssoAvailabilityAlert,
+							"message": "High 5m and 1h error budget burn for RHSSO",
+						},
+						Expr: intstr.FromString(fmt.Sprintf(`
+							sum( sum(rate(haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace='%s', code="5xx"}[5m]))
+								/sum(rate(haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace='%s'}[5m])) > (14.40 * (1-0.99000)))
```

Comment:
99% availability is an ok starting point for this SLO but our SLA is 99.95 and we would want to up this SLO after we see that it is not failing and causing a lot of noise

Reply:
ok, will continue to follow